runtime/macros/src/lib.rs

```diff
@@ -1,140 +1,11 @@
-// based on riscv-rt
-
-#![deny(warnings)]
-
-extern crate proc_macro;
-#[macro_use]
-extern crate quote;
-extern crate core;
-extern crate proc_macro2;
-#[macro_use]
-extern crate syn;
```

Comment:
This file is a copy of `riscv-rt` as the comment says, but it doesn't fit our needs and can be reduced to a much more concise implementation.